lib/Calendar/calendar.dart

```diff
@@ -115,34 +121,158 @@ class CalendarState extends State<Calendar> {
+                      borderRadius: BorderRadius.circular(8),
+                      onTap: () {
+                        setState(() {
+                          source = 'https://localhost:7059/Calendar/welpen';
+                        });
+                        ScaffoldMessenger.of(context).showSnackBar(
+                          const SnackBar(
+                              content: Text('Keuze veranderd naar welpen')),
+                        );
+                      },
+                      child: ClipRRect(
+                        child: Image.asset('../assets/groups/welpen.png',
+                            width: imageSize, height: imageSize),
+                      ),
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Material(
+                    child: InkWell(
+                      borderRadius: BorderRadius.circular(8),
+                      onTap: () {
+                        setState(() {
+                          source =
+                              'https://localhost:7059/Calendar/ZeeVerkenners';
+                        });
+                        ScaffoldMessenger.of(context).showSnackBar(
+                          const SnackBar(
+                              content:
+                                  Text('Keuze veranderd naar zee verkenners')),
+                        );
+                      },
+                      child: ClipRRect(
+                        child: Image.asset('../assets/groups/zee_verkenner.png',
+                            width: imageSize, height: imageSize),
+                      ),
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Material(
+                    child: InkWell(
+                      onTap: () {
+                        setState(() {
+                          source = 'https://localhost:7059/Calendar/matrozen';
+                        });
+                        ScaffoldMessenger.of(context).showSnackBar(
+                          const SnackBar(
+                              content: Text('Keuze veranderd naar matrozen')),
+                        );
+                      },
+                      child: ClipRRect(
+                        child: Image.asset('../assets/groups/matrozen.png',
+                            width: imageSize, height: imageSize),
+                      ),
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Material(
+                    child: InkWell(
+                      borderRadius: BorderRadius.circular(8),
+                      onTap: () {
+                        setState(() {
+                          source = 'https://localhost:7059/Calendar/stam';
```

Comment:
kan dit anders dat localhost er niet in staat? 
als die op een andere port draait werkt het niet meer

Reply:
Hier zullen we later naar moeten kijken met eventueel een config bestand.